Description: Increase test timeout
 Increase test timeout to avoid FTBS on slow architectures
Bug-Debian: https://bugs.debian.org/894987
Forwarded: not-needed
Author: "Manuel A. Fernandez Montecelo" <mafm@debian.org>
Origin: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=894987
--- a/testme.sh
+++ b/testme.sh
@@ -142,7 +142,7 @@
 {
   make clean > /dev/null
   local _timeout=""
-  which timeout >/dev/null && _timeout="timeout --foreground 90"
+  which timeout >/dev/null && _timeout="timeout --foreground 900"
   if [[ "$MAKE_OPTIONS" =~ "tune" ]]
   then
     # "make tune" will run "tune_it.sh" automatically, hence "autotune", but it cannot
